pkg/virt-operator/creation/components/scc.go

```diff
@@ -76,6 +76,7 @@ func NewKubeVirtControllerSCC(namespace string) *secv1.SecurityContextConstraint
 	}
 	scc.AllowedCapabilities = []corev1.Capability{"NET_ADMIN", "SYS_NICE"}
 	scc.AllowHostDirVolumePlugin = true
+	scc.AllowHostNetwork = true
```

Comment:
Why do we need host network?

Reply:
Because we don't want the attachmentPod to use an IP address from the pod network. The attachment pod will never use the network and there is no way to turn off networking on a pod, so instead we give it the host IP.

Reply:
if this permission becomes an issue at some point, maybe there's a way to create a fake cni plugin just for these attachment pods to use.